lm_eval/models/__init__.py

```diff
@@ -15,6 +16,7 @@
     "anthropic": anthropic_llms.AnthropicLM,
     "textsynth": textsynth.TextSynthLM,
     "dummy": dummy.DummyLM,
+    "llama": llama.LlamaLM
```

Comment:
I think it might be good to differentiate this vs other methods of loading llama via a name like `LlamaCppLM` or something like that.

Reply:
Updated to `LlamaCppLM`

Reply:
How about we name this ggml? My understanding is that that's the generic name of the project, with llama.cpp being a llama-specific version.

Reply:
Good point, renamed to ggml. There are several ggml models that do not work with llama.cpp that I'd like this to cover if needed.

Reply:
While llama.cpp is a specific version, it is the Python wrapper to that (llama-cpp-python) providing the API the class uses if I'm not mistaken. GGML doesn't have a generic LLM interface like 🤗 transformers.

Reply:
For the non llama models I'm using ctransformers (https://github.com/marella/ctransformers) and writing my own FastAPI implementations that match the llama-cpp-server spec. https://huggingface.co/spaces/matthoffner/wizardcoder-ggml

Reply:
Yeah I'd like to get to those after the initial llama-cpp-python models. I have some FastAPI examples on my HF (see the wizardcoder-ggml example)